brave/brave-core
# fix(wallet): Solana Swap Transaction Intent

## Description 

Fixes the `Transaction Intent` description for `Solana` swaps and bridges in the `Post Confirmation` panel


Resolves <https://github.com/brave/brave-browser/issues/42516>



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Open the `Wallet` and create a `Solana` swap or bridge transactions
2. Confirm the `Transaction` and wait for the `Post Confirmation` panel to appear
3. The `Transaction Intent` description should not be broken and should display generic messaging.

![image](https://github.com/user-attachments/assets/96aaef09-7366-46bd-ac53-836e476c2fa7) | ![image](https://github.com/user-attachments/assets/775bf1d8-6ad8-4cc0-bbc8-0e15a53d9c98) | ![image](https://github.com/user-attachments/assets/64cfc074-a549-4ac5-a3d3-89c0890eca54)
--|--|--

https://github.com/user-attachments/assets/87735126-44ac-49e0-8a6b-75ef77aff7ab
